ecosystem/typescript/sdk_v2/tests/unit/helper.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+export const ed25519 = {
```

Comment:
this comment is a nit and can be addressed in a later PR. 

The idea was to have a helper file with an Account that we can later use for all tests
```
export const account = {
  privateKey: "0xc5338cd251c22daa8c9c9cc94f498cc8a5c7e1d2e75287a5dda91096fe64efa5",
  publicKey: "0xde19e5d1880cac87d57484ce9ed2e84cf0f9599f12e7cc3a52e4e7657a763f2c",
}
```
don't think all those properties should be under a ed25519. but as mentioned before we can handle it later when we have a complete crypto/account classes

Reply:
This is a little weird, unless we're trying to use these things across multiple places, it probably can be centralized to this test.

Reply:
just to clarify on my comment, the end goal thought is to use something similar to https://github.com/alchemyplatform/alchemy-sdk-js/blob/main/test/test-util.ts where we create an account, this account will submit some transactions and then we tests everything against this account

Reply:
Yeah I think this make sense. Once we have the Account class setup, we can export a static Account object for testing only.